Makefile

```diff
@@ -0,0 +1,35 @@
+MODULE := github.com/mongodb-forks/drone-helm3
+CMD_NAME ?= drone-helm
+VERSION := $(shell grep '^const Version =' internal/version/version.go | cut -d\" -f2)
+
+RUN ?= .*
+PKG ?= ./...
+.PHONY: test
+test: ## Run tests in local environment
+	golangci-lint run --timeout=5m $(PKG)
+	go test -cover -run=$(RUN) $(PKG)
+
+.PHONY: docker
+docker: ## Build local development docker image with cached go modules, builds, and tests
+	@docker build -f build/Dockerfile-test -t $(CMD_NAME):latest .
```

Comment:
```suggestion
	@docker build -f Dockerfile -t $(CMD_NAME):latest .
```

Reply:
the Dockerfile-test is for local development purposes and the Dockerfile is to produce the final artifact.

Reply:
but there is no `build/Dockerfile-test`

Reply:
sorry, didn't notice that ./build was in gitignore, so wasn't pushing my changes there.

I see your point that "build" is probably redundant in terms of development as that's only required when publishing and that will happen in Drone.